pkg/cosign/fulcio/fulcio.go

```diff
@@ -46,6 +47,7 @@ const (
 	FlowNormal = "normal"
 	FlowDevice = "device"
 	FlowToken  = "token"
+	altRoot    = "COSIGN_ROOT"
```

Comment:
nit: maybe "SIGSTORE_ROOT_DIR" instead? These are really the roots for the sigstore project and aren't cosign specific.

Reply:
agree on both, I will add the other flags added too.